staging/src/k8s.io/kubectl/pkg/generate/versioned/run.go

```diff
@@ -87,6 +87,25 @@ func getArgs(genericParams map[string]interface{}) ([]string, error) {
 	return args, nil
 }
 
+// getAnnotations returns map of annotations.
```

Comment:
Is there a way you can use [this](https://github.com/kubernetes/kubernetes/blob/b497fa1d34ec15c4219d591e541c2cf811271357/staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate.go#L380-L453) all here?